crates/tests-e2e/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tests-e2e"
```

Comment:
❓ Why not `e2e-tests`? It sounds more natural.

Reply:
I feel it will be easier to find on disk when I start with its primary purpose - ie. `tests`.

Was thinking of even leaving out `e2e` from the crate name, and just have it in the justfile - I don't think we'll have different set of tests at all.